README.md

```diff
@@ -36,6 +44,10 @@ Return to your original locations.  In this step all groups will act like Design
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+1. We can create an application for your phone or computer that completely blocks access to the distracting websites that prohibit work. Because "these websites" will be dependent on the user and the situation, they will be chosen by the user, and can always be changed later. Although this change feature can be used to procrastinate and re enable access to blocked websites, which is a weakness, we have to assume that because the user has the app (and wants to get work done), they will use it properly. We will have an option to disable access to these sites during a specific time interval as well, if that make the user do more work.
```

Comment:
Web applications that serve this very purpose do exist! You key in on a very important loophole of sorts with these types of applications when you discuss this re-enable function that is always open available to users. Unfortunately like you've said, the creators of such applications can only do so much to limit users while also giving them some control over which sites are blocked and which aren't.